packages/functions/src/cache/index.ts

```diff
@@ -0,0 +1,66 @@
+import { getContext } from '../get-context';
+import { CacheOptions, FunctionCache } from './types';
+import { InMemoryCache } from './in-memory-cache';
+
+const defaultKeyHashFunction = (key: string) => {
+  let hash = 5381;
+  for (let i = 0; i < key.length; i++) {
+    hash = (hash * 33) ^ key.charCodeAt(i);
+  }
+  return (hash >>> 0).toString(); // Convert the hash to a string
```

Comment:
suggestion: Use hexadecimal to save some memory.

```suggestion
  return (hash >>> 0).toString(16); // Convert the hash to a string
```

Reply:
updated